module4/labs/Week1/JavaScript.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Intro to JavaScript Lab
+---
+
+## Lab Objectives
+* Practice using data types, variables, operators, and conditionals in JavaScript
+* Practice writing and calling JavaScript functions
+
+## Gradebook
+
+Your task is to create a gradebook in JavaScript 
+
+* Fork [this repl]() in Replit.
```

Comment:
I really like this structure of a larger project with the initial requirements and then extra challenges.

I hope writing to a file in replit is easy, but seems probably worth trying before students give it a go.